validator/src/main.rs

```diff
@@ -1782,32 +1782,14 @@ pub fn main() {
                 .hidden(true),
         )
         .arg(
-            Arg::with_name("relayer_address")
-                .long("relayer-address")
-                .value_name("relayer_address")
-                .takes_value(true)
-                .help("Address of the relayer")
-        )
-        .arg(
-            Arg::with_name("block_engine_address")
-                .long("block-engine-address")
-                .value_name("block_engine_address")
-                .takes_value(true)
-                .help("Address of the block engine")
+            Arg::with_name("block_engine_url")
```

Comment:
can you change to block_engine_address? then they can just remove one line from startup

Reply:
and can deprecate the old ones

Reply:
nvmd. if remove one line but they keep the port, it'll try to auth through block engine specific port + fail